sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -165,17 +165,28 @@ describe("LocalWorkspace", () => {
         // Adding existing envs should succeed.
         await stack.addEnvironments("automation-api-test-env", "automation-api-test-env-2");
 
+        let envs = await stack.listEnvironments();
+        assert.strictEqual(envs.length, 2);
+        assert.strictEqual(envs[0], "automation-api-test-env");
+        assert.strictEqual(envs[1], "automation-api-test-env-2");
```

Comment:
Similarly, this could be:

```suggestion
        assert.deepStrictEqual(envs, ["automation-api-test-env", "automation-api-test-env-2"]);
```